clinica/utils/io.py

```diff
@@ -162,6 +197,10 @@ def check_caps_folder(caps_directory):
              Fore.BLUE, caps_directory, Fore.RESET)
         )
 
+    if len([item for item in listdir(caps_directory) if item.startswith('sub-')]) > 0:
+        raise ClinicaCAPSError(Fore.RED + '\n[Error] Your CAPS directory contains at least one folder whose name '
```

Comment:
Should we display what we found? In the case that the use created/moved folder accidentally such that we have e.g. `<CAPS>/sub-CLNC01`? :thinking: 

If you consider this useless, you can ignore my comment :-)

Reply:
Perfect!